cocos/3d/CCAnimate3D.h

```diff
@@ -91,32 +111,88 @@ class CC_DLL Animate3D: public ActionInterval
     
     virtual void update(float t) override;
     
-    /**get & set speed, negative speed means playing reverse */
+    /**
+     * @~english Get speed, negative speed means playing reverse 
+     * @~chinese 获取播放速度，负数的速度意味着动画正在倒放
+     * @return @~english The speed
+     * @~chinese 速度
+     */
     float getSpeed() const;
+
+    /**
+    * @~english Set speed, negative speed means playing reverse
+    * @~chinese 设置播放速度，负数的速度意味着动画将会倒放
+    * @param speed @~english the speed
+    * @~chinese 速度
+    */
     void setSpeed(float speed);
     
-    /**get & set blend weight, weight must positive*/
+    /**
+     * Get the blend weight
+     * @~chinese 获取动画混合权重
+     * @return the blend weight
+     * @~chinese 
+     */
     float getWeight() const { return _weight; }
+
+    /**
+     * @~english Set the blend weight
+     * @~chinese 设置动画混合权重
+     * @param weight @~english The blend weight, the blend weight must be positive
+     * @~chinese 权重，权值必须为正
+     */
     void setWeight(float weight);
     
-    /**get & set origin interval*/
+    /**
+     * @~english Set origin interval
```

Comment:
It is internally used currently. For example, Sprite3D is playing an Animate3D, and then another Animate3D is going to be played. This function will be used to make the transition from on Animate3D to another smoothly.